File: app/src/main/java/com/websarva/wings/android/kaigonote/HiyariActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.HiyariBinding;

public class HiyariActivity extends AppCompatActivity {
    private HiyariBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = HiyariBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/HiyariActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.HiyariBinding

class HiyariActivity : AppCompatActivity() {
    private var binding: HiyariBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = HiyariBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}